.circleci/config.yml

```diff
@@ -281,24 +281,28 @@ jobs:
             DIRS_TO_CHECK="${PWD}/packages/celotool,${PWD}/packages/protocol"
             ./scripts/ci_check_if_test_should_run_v2.sh ${DIRS_TO_CHECK}
       - run:
-          name: Setup Go language
+          name: Setup languages
```

Comment:
Would it make sense to split this into two separate tasks, one for each language?
If so, ditto elsewhere.